components/script/dom/htmlmediaelement.rs

```diff
@@ -301,6 +301,10 @@ impl From<MediaStreamOrBlob> for SrcObject {
     }
 }
 
+fn now() -> SystemTime {
+    SystemTime::now()
+}
+
```

Comment:
I think in this case you should omit the helper function entirely as it just calls a single function. It's more useful for the reader to know what this does.